Instruments/bridge12.py

```diff
@@ -490,6 +490,7 @@ def soft_shutdown(self):
         """do everything to shut the B12 down, but don't break out of the
         with block or close the USB connection"""
         self.set_power(0)
+        self.set_amp(False)
```

Comment:
1) we want to test this
2) you do need a scheme for turning it back on --> you point out dip lock should turn it on **so** you just need to set the attribute that says dip lock has been run to False